engine/kube/util.go

```diff
@@ -7,12 +7,14 @@ package kube
 import (
 	"path"
 	"path/filepath"
-	"strings"
 	"strconv"
+	"strings"
 
 	"github.com/drone/drone-runtime/engine"
+	"github.com/drone/drone/cmd/drone-controller/config"
```

Comment:
this is a standalone package and should not link to the main Drone repository as a dependency.

Reply:
I was wondering about this. I couldn't find a better way to pull the config in without copy-pasting the config code.. :/

Reply:
you would typically pass these through the Engine Constructor from Drone https://github.com/drone/drone-runtime/blob/master/engine/kube/kube.go#L34

I recommend looking at how the controller passes the Node to the Engine, for example.

Reply:
Thanks for the tip. I will work on this tonight

Reply:
This, too has been removed in recent commits!